pkg/apis/operator.min.io/v1/helper.go

```diff
@@ -100,17 +101,19 @@ func (mi *MinIOInstance) AutoCert() bool {
 // VolumePath returns the paths for MinIO mounts based on
 // total number of volumes per MinIO server
 func (mi *MinIOInstance) VolumePath() string {
-	if mi.Spec.VolumesPerServer == 1 {
+	v, _ := strconv.Atoi(mi.Spec.VolumesPerServer)
```

Comment:
We should not ignore the error here. Customers could be hand crafting the CRD.

Reply:
validation is done initially here, https://github.com/minio/minio-operator/pull/169/files#diff-5fba95502c597c71c1ec5de3290ef01bR453 while someone submits a MinIO Tenant Yaml.